packages/@o3r/testing/package.json

```diff
@@ -120,6 +125,7 @@
     "tslib": "^2.4.1"
   },
   "devDependencies": {
+    "@amadeus-it-group/kassette": "^1.6.0",
```

Comment:
I wonder if it should not be in optional peer as  well.
In case of `skipLibCheck: false`  the dependency will be mandatory by the caller and it will need to be in this version range